packages/editor/src/components/entities-saved-states/style.scss

```diff
@@ -1,11 +1,70 @@
-.editor-entities-saved-states__save-button {
-	display: block;
-	margin-left: auto;
-	margin-right: 0;
-}
-.editor-entities-saved-states__entity-type-list {
-	h2 {
-		font-size: 18px;
-		margin: 20px 0 10px;
+.entities-saved-states__panel {
+	@include reset;
```

Comment:
Do you recall why we need a "reset" here. Seems weird since this is meant for global resets.

Reply:
Ideally, components don't depend on global resets though.

Reply:
It may not be necessary.  I think I added it b/c post-publish-panel was using it at the time as well and I was trying to ensure the styles were the same between the two.  It looks like it's not there anymore either.